products/idn/docs/identity-now/rules/index.md

```diff
@@ -0,0 +1,175 @@
+---
+id: rules
+title: Rules
+pagination_label: Rules
+sidebar_label: Rules
+sidebar_position: 2
+sidebar_class_name: rules
+keywords: ["rules"]
+description: Rule guide for developing rules in Identity Now.
```

Comment:
I wouldn't use the word _guide_ here, as we use Guides to define guides/tutorials/walkthroughs on how to do something. I'd say _Documentation_ instead.

Reply:
Updated.